docs/website/components/Header.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <header id="header" class="flex flex-wrap">
-    <div class="flex items-center justify-start max-w-6xl mx-auto py-2">
+  <header id="header" class="flex flex-wrap w-4/5 mx-auto">
+    <div class="flex py-2">
       <Logo></Logo>
     </div>
-    <div class="flex items-center justify-end max-w-6xl mx-auto py-6">
+    <div class="flex py-6 ml-auto">
```

Comment:
Not sure which change did it, but seeing this:

<img width="1680" alt="Screen Shot 2019-10-31 at 10 23 30 AM" src="https://user-images.githubusercontent.com/3383143/67970715-aca47200-fbc8-11e9-8cb3-06b631427fcc.png">

Notice the bottom border doesn't extend the full width of the screen.

Reply:
Oh yeah, good catch. Let me try to fix this.